docs/cpu_templates/schema.json

```diff
@@ -5,6 +5,32 @@
     "description": "Custom CPU template language description",
     "type": "object",
     "properties": {
+        "kvm_capabilities": {
+            "type": "array",
+            "items": {
+                "description": "Additional KVM capabilities can be added or existing (built in) capabilities can be removed from the firecracker checks. To add KVM capability to the checklist specify decimal number of the corresponding KVM capability. To remove a KVM capability from the checklist specify decimal number of the corresponding KVM capability with '!' mark in the front. Works on both x86_64 and aarch64.",
+                "type": "string",
+                "examples": ["171", "!172"]
+            }
+        },
+        "vcpu_features": {
+            "type": "array",
+            "items": {
+                "description": "vCPU features to enable during vCPU initialization. Only for aarch64.",
+                "type": "object",
+                "properties": {
+                  "index": {
+                        "description": "Index into kvm_vcpu_init::features array. Current linux kernel (6.4.10) only allows value 0.",
```

Comment:
a bit worried that "Current" is not going to be true at some point, and we will not be updating this doc very often to keep it up to date.

Reply:
Well, we specify the kernel version here, so even if we will not update the doc, it will still be technically valid.

Reply:
my point being, it will no longer be the "current", which is not very nice.

Reply:
Ok, how do you want to fix this?

Reply:
we can say something like "As of Linux kernel 6.4, only value 0 is allowed"

Reply:
Ok done.